nano/node/bootstrap/bootstrap_server.cpp

```diff
@@ -95,7 +89,9 @@ std::size_t nano::bootstrap_listener::connection_count ()
 
 void nano::bootstrap_listener::accept_action (boost::system::error_code const & ec, std::shared_ptr<nano::socket> const & socket_a)
 {
-	if (!node.network.excluded_peers.check (socket_a->remote_endpoint ()))
+	auto const remote_endpoint = socket_a->remote_endpoint ();
+	debug_assert (remote_endpoint.has_value ());
```

Comment:
Can we guarantees this? If they connect and immediately disconnect before this gets processed?